nikola/plugin_categories.py

```diff
@@ -493,6 +493,17 @@ class Taxonomy(BasePlugin):
         in the metadata. If set to None, the classification won't be stored
         in the metadata.
 
+    metadata_friendly_name = None:
+        The name to be used when storing the friendly names of the classification
+        in the metadata. If set to None, the friendly names won't be stored
```

Comment:
Then how does a plugin pass language-dependent information to a post object? By adding stuff to the post object itself? I don't know of any other good way than modifying `post.meta`.